### PR TITLE
Extensibility

### PR DESCRIPTION
**Pull request by zionrubin**

### CODE REVIEW

File: liminal/settings.py [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+import os
+import sys
+
+from liminal.core.environment import get_liminal_home
+
+LIMINAL_HOME = get_liminal_home()
+
+
+def make_plugin_dirs():
+    plugins = os.path.join(LIMINAL_HOME, 'liminal/plugins')
```
Comment (aviemzur) [inline liminal/settings.py:27]: Why `liminal/plugins`? Can we make it just `plugins`?

Comment (zionrubin) [reply liminal/settings.py:27]: sure

**Comment by assapin** [comment]
I would add a short guide + example for how to add plugin of each type before merging this as "done".

**Review by assapin — APPROVED** [review]

File: scripts/liminal [hunk 2]
```diff
@@ -141,7 +143,9 @@ def deploy_liminal_apps(path, clean):
     liminal_home = environment.get_liminal_home()
     os.makedirs(liminal_home, exist_ok=True)
     os.makedirs(environment.get_dags_dir(), exist_ok=True)
+
     deploy_liminal_core_internal(liminal_home, clean)
+    deploy_plugins(liminal_home)
```
[the remark this answers is not in the document]
Comment (zionrubin) [reply scripts/liminal]: looks like you checked the wrong version (not sure how 😟 )

**Pull request merged**